src/azure-cli/azure/cli/command_modules/resource/custom.py

```diff
@@ -428,70 +649,45 @@ def _deploy_arm_template_subscription_scope_unmodified(cli_ctx,
         template_content = read_file_content(template_file)
         template_obj = _remove_comments_from_json(template_content)
 
+    if rollback_on_error == '':
```

Comment:
>if rollback_on_error == '' [](start = 4, length = 26)

this logic seems a little tricky, what about if rollback_on_error is None, what not just check if not rollback_on_error which means both Non and empty string

Reply:
I'm not sure about this one. This part was not changed. it's existing code.